app/index.js

```diff
@@ -40,21 +40,49 @@ if (!gotTheLock) {
 			notifications.addDesktopNotificationHack(iconPath);
 		}
 
-		window.webContents.on('new-window', (event, url, frame, disposition) => {
+		window.webContents.on('new-window', (event, url, frame, disposition, options) => {
+			console.debug('DEBUG - new-window triggered')
 			if (url.startsWith('https://teams.microsoft.com/l/meetup-join')) {
 				event.preventDefault();
 				window.loadURL(url);
 			} else if (disposition !== 'background-tab') {
 				event.preventDefault();
-				shell.openExternal(url);
+				if ((url === 'about:blank')) {
+					// Create a new hidden window to load the request in the background
+					console.debug('DEBUG - captured about:blank');
+					const win = new BrowserWindow({
+						webContents: options.webContents, // use existing webContents if provided
+						show: false
+					})
+					options.webContents.session.webRequest.onBeforeRequest((details, callback) => {
+						domain = details.url.match(/:\/\/(.[^/]+)/)[1]
+						if (/^.*(microsoft|outlook|skype|urlp\.sfbassets).*$/.test(domain)) {
```

Comment:
This seems to successfully grab the domains used in the Teams app but of course this can change so I'm open to a better solution.

Reply:
Hi @HarlemSquirrel , I can't test this at the moment (out of office) but maybe add this as a config option. That way we don't need to release a new build if the URLs change. Cheers